### ISSUE TITLE
Build error as secondary dependency of printpdf

### ISSUE THREAD

**Issue by qtfkwk** [time-rs/time#263]
```
error: expected an item keyword
   --> /home/qtfkwk/.cargo/registry/src/github.com-1ecc6299db9ec823/time-0.2.8/src/utc_offset.rs:366:13
    |
366 |             let tm = timestamp_to_tm(datetime.timestamp())?;
    |             ^^^
    |
   ::: /home/qtfkwk/.cargo/registry/src/github.com-1ecc6299db9ec823/cfg-if-0.1.9/src/lib.rs:41:40
    |
41  |         if #[cfg($($meta:meta),*)] { $($it:item)* }
    |                                        -------- while parsing argument for this `item` macro fragment

error: aborting due to previous error

error: could not compile `time`.
```

This is relatively consistent on both a Windows 10 Debian WSL and a Linux laptop, both with Rust 1.44 stable. I say relatively because for some reason the build succeeds somehow sometimes, but no idea why.

**Comment by qtfkwk** [comment]
See also https://github.com/fschutt/printpdf/issues/55

**Comment by qtfkwk** [comment]
`cargo update` seems to resolve the issue... will continue testing and share any additional observations.

**Comment by jhpratt** [comment]
Duplicate of #233, which was fixed in #234 and [released on 2020-03-13](https://github.com/time-rs/time/blob/master/CHANGELOG.md#029-2020-03-13).
